src/main/java/com/google/sps/servlets/ImageHandlerServlet.java

```diff
@@ -67,7 +80,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     try {
       String imageBlobKeyString = createUploadedBlobKey(request, IMAGE_PARAMETER);
       // create and store store current player
```

Comment:
// create a new player from the currently logged in user and store it in PlayerDatabase